doc/pages/scopes.asciidoc

```diff
@@ -19,7 +19,16 @@ These scoped objects are:
 Scopes are named as follows:
 
 *window*::
-    context linked to the window displaying a buffer
+    context linked to the window displaying a buffer.
```

Comment:
The description that clicked for me was "context of a particular client viewing a particular buffer".